api/provider/aws/environment_delete.go

```diff
@@ -11,14 +11,15 @@ import (
 )
 
 func (e *EnvironmentProvider) Delete(environmentID string) error {
-	//todo: use conifig.Instance()
-	fqEnvironmentID := addLayer0Prefix("INSTANCE", environmentID)
+	fqEnvironmentID := addLayer0Prefix(e.Config.Instance(), environmentID)
 
-	if err := e.deleteASG(fqEnvironmentID); err != nil {
+	autoScalingGroupName := fqEnvironmentID
```

Comment:
This is a good way of indicating that fqEnvironmentID maps to asg name, launch config. 👍